spec/commands/binstubs_spec.rb

```diff
@@ -145,6 +145,21 @@
     end
   end
 
+  context "--standalone" do
```

Comment:
I think this context could be more specific/accurate. Something along the lines of "after installing with --standalone"? You could also move the `install_gemfile` and `bundle "install --standalone"` invocations to a before block that goes with the context then.